content/en/blog/2025/ambient-performance/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Istio: the most performant way to achieve network security"
+description: Istio's' ambient mode trailblazes as the highest performance security solution in the ecosystem.
```

Comment:
```suggestion
title: "Istio: The Highest-Performance Solution for Network Security"
description: Ambient mode provides more encrypted throughput than any other project in the Kubernetes ecosystem.
```

Reply:
change the idiom a little.  Open to changing the description back; just didn't want to say "performance" twice.